infra-as-code/bicep/orchestration/hubPeeredSpoke/README.md

```diff
@@ -0,0 +1,125 @@
+# Module: orchestration\hubPeeredSpoke - Spoke network, including peering to Hub
+
+This module acts as an orchestration module that create and configures a spoke network to deliver the Azure Landing Zone Hub & Spoke architecture which is also described in the wiki on the [Deployment Flow article](https://github.com/Azure/ALZ-Bicep/wiki/DeploymentFlow).
+
+Module deploys the following resources:
+
+- Subscription placement in Management Group hierarchy - if parPeeredVnetSubscriptionMGPlacement is specified
+- Virtual Network (Spoke VNet)
+- UDR - if parNextHopIPAddress and resource id of hub virtual network object is specified
+- Hub to Spoke peering - if resource id of hub virtual network object is specified in parHubVirtualNetworkID
+- Spoke to hub peering - if resource id of hub virtual network object is specified in parHubVirtualNetworkID
+- Spoke to virtual WAN peering - if resource id of virtual WAN hub object is specified in parHubVirtualNetworkID
+
```

Comment:
Can we add a note here that you can only choose the traditional Hub & Spoke method OR VWAN and not both?

Just to avoid confusion where someone tries to do traditional peering and also VWAN connection to a hub?

Reply:
Updated in newest commit